packages/components/src/components/progress-bar/progress-bar.tsx

```diff
@@ -9,20 +9,24 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-import { Component, Prop, h, Host } from '@stencil/core';
+import { Component, Prop, h, Host, Element } from '@stencil/core';
 import classNames from 'classnames';
+import statusNote from '../../utils/status-note';
+
 let i = 0;
 @Component({
   tag: 'scale-progress-bar',
   styleUrl: './progress-bar.css',
   shadow: true,
 })
 export class ProgressBar {
+  /* Host HTML Element */
+  @Element() hostElement: HTMLElement;
   /** (optional) Progress bar busy switch */
   @Prop() busy?: boolean = false;
   /** (required) Progress bar percentage */
   @Prop() percentage: number = 0;
-  /** (optional) Progress bar customColor */
+  /** DEPRECATED - (optional) Progress bar customColor */
```

Comment:
I would use the JSDoc [`@deprecated`](https://jsdoc.app/tags-deprecated.html) syntax here…